src/audio/src/src.c

```diff
@@ -657,14 +657,13 @@ static int src_params(struct comp_dev *dev,
 
 	comp_info(dev, "src_params()");
 
+	src_set_params(dev, params);
```

Comment:
why set before verify?

Reply:
param is figured out by module config so we need to first set up param then verify param.
For IPC3 path, the param is sent by host driver

Reply:
@RanderWang best to state as a comment so that it's easy to follow.

Reply:
@RanderWang best to state as a comment so that it's easy to follow.

Reply:
sure, updated